OG_Format.adoc

```diff
@@ -107,7 +107,6 @@ https://vocab.nerc.ac.uk/collection/L06/current/[_https://vocab.nerc.ac.uk/colle
 
  |mandatory |“Autonomous Underwater Vehicle”
 |platform_vocabulary |Controlled vocabulary for the names used in the "platform" attribute. |mandatory |https://vocab.nerc.ac.uk/collection/L06/current/27/[_https://vocab.nerc.ac.uk/collection/L06/current/27/_]
-|wmoid |Wmo identifier |mandatory |
 |id a|
```

Comment:
at the moment it will remove the wmoid to be mandatory, but if I understand you want to keep it mandatory. please clarify!

Reply:
Mandatory as global attribute is removed, but mandatory as variable remains! We don't need both and are in agreement to keep only as variable.